doc/api/resources/variations.rst

```diff
@@ -0,0 +1,256 @@
+Variations
```

Comment:
We could call this page ``Item variations`` to make it clearer to see the relationship in the table of contents.